smart-contracts/test/Unlock/upgradeLock.js

```diff
@@ -130,3 +130,114 @@ describe('upgradeLock (deploy template with Proxy)', () => {
     assert.equal(await lock.sayHello(), 'hello world')
   })
 })
+
+
+describe('upgrades', async () => {
```

Comment:
I added this which deploys a lock v9 and upgrades consecutively... It does not test everything (we can add later more things) but it currently fails, which helps debug.